Team packets decoding fails on 1.21.5

This issue only arises with specific setups.

**Steps for reproduction**:
- velocity 3.4.0-496
- BTLP-velocity 1162 snapshot
- paper 1.21.4-227 backend
- ViaVersion 5.3.3-411 on backend
- AnimatedScoreBoard 0.3.7 on backend
- Join in 1.21.5
- Get kicked with "Unable to connect you to [server]. Please try again later".

1.21.4+via is probably not relevant to the problem.

https://paste.helpch.at/icanuqemiq.md
The ASB stacktrace is not always present. This problem doesn't happen if I remove BTLP.

ASB is not open, but it's not obfuscated. As far as I've seen, the null object is either null at init or get set null by `PlayerScoreboard#remove` which is called in player quit AND kick listeners. So maybe it gets called twice sometimes, generating the NPE, and the stracktrace probably unrelated.

The Velocity log is self explanatory, mainly:
`io.netty.handler.codec.CorruptedFrameException: Error decoding class codecrafter47.bungeetablistplus.protocol.Team Direction CLIENTBOUND Protocol 1.21.5 State PLAY ID 67`
`Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 7 out of bounds for length 5
        at codecrafter47.bungeetablistplus.protocol.Team.decode(Team.java:75) ~[?:?]`

--- a/velocity/src/main/java/codecrafter47/bungeetablistplus/util/ReflectionUtil.java
+++ b/velocity/src/main/java/codecrafter47/bungeetablistplus/util/ReflectionUtil.java
@@ -26,21 +26,7 @@ import java.lang.reflect.Field;
 import java.lang.reflect.Method;
 import java.util.function.Supplier;
 
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_12;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_12_1;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_13;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_14;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_15;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_17;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_19_1;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_19_3;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_19_4;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_20_2;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_20_3;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_20_5;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_21_2;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_8;
-import static com.velocitypowered.api.network.ProtocolVersion.MINECRAFT_1_9;
+import static com.velocitypowered.api.network.ProtocolVersion.*;
 
 public class ReflectionUtil {
 
@@ -90,7 +76,8 @@ public class ReflectionUtil {
                         (StateRegistry.PacketMapping) packetMapping.newInstance(0x5C, MINECRAFT_1_20_2, null, false),
                         (StateRegistry.PacketMapping) packetMapping.newInstance(0x5E, MINECRAFT_1_20_3, null, false),
                         (StateRegistry.PacketMapping) packetMapping.newInstance(0x60, MINECRAFT_1_20_5, null, false),
-                        (StateRegistry.PacketMapping) packetMapping.newInstance(0x67, MINECRAFT_1_21_2, null, false)
+                        (StateRegistry.PacketMapping) packetMapping.newInstance(0x67, MINECRAFT_1_21_2, null, false),
+                        (StateRegistry.PacketMapping) packetMapping.newInstance(0x66, MINECRAFT_1_21_5, null, false)
                 });
                 return;
             } catch (Exception e) {

